app/policies/repository_policy.rb

```diff
@@ -22,6 +22,7 @@ def resolve
   end
 
   def show?
+    return false unless resource
```

Comment:
Please add a (policy) spec for this case.

Reply:
Done in b23173f